book/website/reproducible-research/code-quality/code-quality-readability.md

```diff
@@ -112,17 +112,17 @@ These are of course just guidelines, and you should choose elements that suit yo
 However, and again, it is important to ensure that you are consistent when collaborating, and can agree on a common style.
 It could be useful to create a readme file describing your coding style so collaborators or contributors can follow your lead.
 
-### ...end. ...end.  ...or end.\\n
+## ...end. ...end.  ...or end.\\n
 
 If you are sharing text files or working collaboratively on manuals or documents, then there is a lot of controversy surrounding whether to use one or two spaces after a period.
 When using {term}`Markdown`, it can be clearer to include a new line after every sentence.
 This chapter (and most, if not all, of this book) has a new line after every sentence that makes the raw text easier to read, review and solve the spacing issue.
 
-```{figure} ../../../../figures/xkcd1285.png
+```{figure} ../../../figures/xkcd1285.png
```

Comment:
I think it would be better to get this by URL, rather than have a copy in the repository.

Reply:
Where is the image stored?